invenio_theme/assets/semantic-ui/js/invenio_theme/truncate.js

```diff
@@ -0,0 +1,38 @@
+import $ from "jquery";
```

Comment:
please import it from `theme.js` or move it there so that it will be part of the base JS.